package.json

```diff
@@ -81,5 +82,10 @@
       "eslint --fix",
       "git add"
     ]
+  },
+  "prettier": {
+    "printWidth": 500,
```

Comment:
Thought e we were going for 120?

Reply:
I was just replicating `.editorconfig` settings:
https://github.com/HerbCaudill/coronadatascraper/blob/c64cf9c506b445e34a0c11776874b357ad94bf52/.editorconfig#L3

Happy to change it in both places if that's what we want, lmk.

Reply:
Not sure where we landed. @lazd ?

Reply:
I see now that .editorconfig changed to 120 - I've updated the prettier config.